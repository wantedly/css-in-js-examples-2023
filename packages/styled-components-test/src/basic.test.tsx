import { render } from "@testing-library/react";
import React from "react";
import { test, expect } from "@jest/globals";

import App from "./basic";

test("snapshot", () => {
  render(<App />);
  expect(document.children[0].innerHTML).toMatchSnapshot();
});
