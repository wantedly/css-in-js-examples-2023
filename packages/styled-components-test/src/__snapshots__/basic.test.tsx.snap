// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `"<head><style data-styled="active" data-styled-version="6.1.4">.bmcGJu{background:red;}</style></head><body><div><button class="basic__Button-sc-w201kt-0 bmcGJu">Test</button></div></body>"`;
